README.md

```diff
@@ -48,8 +48,8 @@
 	- OS
 		- [Operating Systems](#os)
 	- Gaming
-		- [Bindings](#game-bindings)
-		- [Game Engine Bindings](#game-engine-bindings)
+		- [Game and Graphics Bindings](#game-bindings)
```

Comment:
You need to also change the header in the content if you change the ToC

If you change the header, add an `<a name...` tag like occuring elsewhere in the document, to give backwards compatibility for old links.

Reply:
still needs change